src/libs/actions/App.ts

```diff
@@ -136,6 +136,13 @@ Onyx.connect({
     },
 });
 
+let allReportsDraftComment: Record<string, string | undefined> = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT,
+    waitForCollectionCallback: true,
+    callback: (value) => (allReportsDraftComment = value),
+});
+
```

Comment:
I believe we will remove Onyx completely soon and avoid using it anywhere inside the App. Is there any alternative solution for it?

Reply:
If that's the only way then we should do it.